catalog/CatalogTypedefs.hpp

```diff
@@ -36,6 +36,10 @@ typedef int attribute_id;
 typedef std::size_t partition_id;
 typedef int numa_node_id;
 
+// Catalog ids use negative values as invalid ids. Mark -1 as constant invalid
+// id for the catalog ids.
+constexpr int kInvalidCatalogId = -1;
```

Comment:
I have 2 questions:
1-) Should I also delete my comment?
2-) Should I change the copyright?